extensions/amp-subscription/0.1/amp-subscription.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {installStylesForDoc} from '../../../src/style-installer';
+import {SubscriptionPlatform} from './amp-subscription-platform';
+
+/** @const */
+const TAG = 'amp-subscription';
+
+export class SubscriptionSubscription {
+  /**
+   * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+   */
+  constructor(ampdoc) {
```

Comment:
Both, service and platform classes need tests created right away. Obviously not a lot will be there, but some small pieces will be.

Reply:
Done